diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3b75cad..cdd056f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,4 +26,5 @@ add_executable(plasma-theme
         utils/signals.cpp
         utils/signals.h)
 
-target_link_libraries(plasma-theme KF5::ConfigCore)
\ No newline at end of file
+target_link_libraries(plasma-theme KF5::ConfigCore)
+install(TARGETS plasma-theme DESTINATION bin/)
